### PR TITLE
Add emacs theme

### PR DESCRIPTION
**Pull request by Yevgnen**
This commit adds a complete clone of the Emacs default theme. It is based on the default Emacs settings and the default colors of the `emacs-tree-sitter` package.

Screenshot:
Screen Shot 2022-08-13 at 16 35 46

Origin:
Screen Shot 2022-08-13 at 23 55 26

### CODE REVIEW

File: runtime/themes/emacs.toml [hunk 1]
```diff
@@ -0,0 +1,636 @@
...
+"ui.menu" = { fg = "black", bg = "cornsilk" }
+"ui.menu.selected" = { fg = "dark_red", bg = "light_blue" }
+"ui.selection" = { bg = "lightgoldenrod1" }
+"ui.selection.primary" = { bg = "lightgoldenrod2" }
+"ui.virtual.whitespace" = "highlight"
+"ui.virtual.ruler" = { bg ="gray95" }
+
+# Disabled by default
+# "ui.cursorline.primary" = { bg = "darkseagreen2" }
+# "ui.cursorline.secondary" = { bg = "darkseagreen2" }
```
Comment (the-mikedavis) [inline runtime/themes/emacs.toml:72]: These are behind a configuration option so they don't need to be commented out to not use them

Comment (Yevgnen) [reply runtime/themes/emacs.toml]: Done!

**Comment by Yevgnen** [comment]
Is there a way to customize the underline colors? The Emacs default looks like

**Comment by the-mikedavis** [comment]
> Is there a way to customize the underline colors?

Not currently but there's a PR https://github.com/helix-editor/helix/pull/3015#issuecomment-1211856562

**Review by the-mikedavis — APPROVED** [review]
This looks nice, thank you! 🎨

**Comment by the-mikedavis** [comment]
Could you add a screenshot to the themes wiki? https://github.com/helix-editor/helix/wiki/Themes

**Comment by Yevgnen** [comment]
@the-mikedavis Done!

**Pull request merged**